### PR TITLE
update base image in Dockerfile

### PR DESCRIPTION
**Pull request by Madhu-1**
As we have the octopus as the latest release base image, this PR updates the base image in Dockerfile

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

File: deploy/cephcsi/image/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ceph/ceph:v14.2
+FROM ceph/ceph:v15.2
```
Comment (nixpanic) [inline deploy/cephcsi/image/Dockerfile]: Why stick to .2 and not use the latest stable v15?

Comment (Madhu-1) [reply deploy/cephcsi/image/Dockerfile]: done updated to v15

File: scripts/Dockerfile.devel [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
 RUN yum -y install \
 	golang \
+	git \
 	make \
 	librados-devel \
 	librbd-devel \
-    && yum -y update \
-    && true
+	&& yum -y update \
```
Comment (nixpanic) [inline scripts/Dockerfile.devel]: why did you change the indention? This removes the distinction between the commands. Please correct the indention again

Comment (Madhu-1) [reply scripts/Dockerfile.devel]: it was by mistake fixed now. PTAL

**Comment by humblec** [comment]
@Madhu-1 may be we can squash the commits and mention the changes together in the text of the commit ?

**Comment by Madhu-1** [comment]
> @Madhu-1 may be we can squash the commits and mention the changes together in the text of the commit ?

squashed 2 and 3rd commit

File: scripts/Dockerfile.devel [hunk 3]
```diff
@@ -1,9 +1,11 @@
-FROM ceph/ceph:v14
+FROM ceph/ceph:v15
 
-ENV GOPATH=/go
+ENV GOPATH=/go \
+	GO111MODULE=on
```
Comment (humblec) [inline scripts/Dockerfile.devel:4]: Is it this made `on` for `go mod` dependency to work while building ? @Madhu-1

Comment (humblec) [inline scripts/Dockerfile.devel:4]: Is it this made `on` for `go mod` dependency to work while building ? @Madhu-1

Comment (Madhu-1) [reply scripts/Dockerfile.devel:4]: Yes humble
